src/input/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! Input processing functionality for Sprocket.
+
+pub mod yaml;
+
+// Re-export the main functionality from the yaml module
+pub use yaml::InputFormat;
+pub use yaml::get_json_file_path;
+pub use yaml::get_json_string;
+pub use yaml::parse_input_file;
```

Comment:
```suggestion
pub use yaml::get_json_file_path;
```
Just export what is being used.